lib/perl/Genome/File/Vcf/Entry.pm

```diff
@@ -592,7 +592,8 @@ sub add_format_field {
     else {
         my $idx = scalar @{$self->{_format}};
         push @{$self->{_format}}, $field;
-        $self->{_format_key_to_idx}{$field} = $idx;
+        my $format_idx = $self->format_field_index;
+        $self->{_format_key_to_idx} = {%$format_idx, $field => $idx};
```

Comment:
Could this be more understandable written as:
```
$format_idx->{ $field } = $idx;
 $self->{_format_key_to_idx} = $format_idx;
```
It also avoids copying the hash into another when using %$format_idx.